syntaxes/src/expression/grammar.ts

```diff
@@ -0,0 +1,886 @@
+    ngExpression: {
+      name: 'meta.expression.ng',
+      patterns: [
+        {
+          include: '#tsString',
+        },
+        {
+          include: '#literal',
+        },
+        {
+          include: '#ternaryExpression',
+        },
+        {
+          include: '#expressionOperators',
+        },
+        {
+          include: '#functionCall',
+        },
+        {
+          include: '#identifiers',
+        },
+        {
+          include: '#parenExpression',
+        },
+        {
+          include: '#punctuationComma',
+        },
+        {
+          include: '#punctuationAccessor',
+        },
+      ],
+    },
+
+    arrayLiteral: {
+      name: 'meta.array.literal.ts',
+      begin: /\[/,
+      beginCaptures: {
+        '0': {
+          name: 'meta.brace.square.ts',
+        },
+      },
+      end: /\]/,
+      endCaptures: {
+        '0': {
+          name: 'meta.brace.square.ts',
+        },
+      },
+      patterns: [
+        {
+          include: '#ngExpression',
+        },
+        {
+          include: '#punctuationComma',
+        },
+      ],
+    },
+
+    booleanLiteral: {
+      patterns: [
+        {
+          name: 'constant.language.boolean.true.ts',
+          match: /(?<!\.|\$)\btrue\b(?!\$)/,
+        },
+        {
+          name: 'constant.language.boolean.false.ts',
+          match: /(?<!\.|\$)\bfalse\b(?!\$)/,
+        },
+      ],
+    },
+
+    expressionOperators: {
+      patterns: [
+        {
+          match: /((?<!\|)(\|)(?!\|))\s?([a-zA-Z0-9\-\_]*)/,
+          captures: {
+            '2': {
+              name: 'keyword.operator.logical.ts',
+            },
+            '3': {
+              name: 'entity.name.function.pipe.ng',
+            },
+          },
+        },
+        {
+          name: 'storage.type.ts',
+          match: /(?<!\.|\$)\b(let)\b(?!\$)/,
+        },
+        {
+          name: 'keyword.control.flow.ts',
+          match: /(?<!\.|\$)\b(await)\b(?!\$)/,
+        },
+        {
+          name: 'keyword.operator.expression.delete.ts',
+          match: /(?<!\.|\$)\bdelete\b(?!\$)/,
+        },
+        {
+          name: 'keyword.operator.expression.in.ts',
+          match: /(?<!\.|\$)\bin\b(?!\$)/,
+        },
+        {
+          name: 'keyword.operator.expression.of.ts',
+          match: /(?<!\.|\$)\bof\b(?!\$)/,
+        },
+        {
+          name: 'keyword.control.if.ts',
+          match: /(?<!\.|\$)\bif\b(?!\$)/,
+        },
+        {
+          name: 'keyword.control.else.ts',
+          match: /(?<!\.|\$)\belse\b(?!\$)/,
+        },
+        {
+          name: 'keyword.control.then.ts',
+          match: /(?<!\.|\$)\bthen\b(?!\$)/,
+        },
+        {
+          name: 'keyword.operator.expression.instanceof.ts',
+          match: /(?<!\.|\$)\binstanceof\b(?!\$)/,
+        },
+        {
+          name: 'keyword.operator.new.ts',
+          match: /(?<!\.|\$)\bnew\b(?!\$)/,
+        },
+        {
+          include: '#typeofOperator',
```

Comment:
Consider putting this at the bottom or top of all the patterns included in `expressionOperators`

Reply:
Changed to the end of the array